services/pafs/README.md

```diff
@@ -0,0 +1,33 @@
+# Submit your flood or coastal erosion risk management project proposal
+
+This service is more commonly known by its original name Project Application and Funding Service (PAFS).
+
+To quote the description on the main project README
+
+> [PAFS] is used by regional management authorities to apply for funding for flood and coastal risk management projects.
+
+The [main entry page](https://www.gov.uk/guidance/flood-and-coastal-defence-funding-submit-a-project) for the service specifies you can use tghe service to
+
+- submit a project proposal
+- update an existing project
+- view the status of a proposal
+
+The one difference to other services is that you first must request an account before you can sign in and submit a project proposal. This is also done online via the service.
+
+## High level overview
+
+The service is made up of a external facing Rails web app, and an internal facing Rails web app. Common functionality is maintained in an engine mounted by both web apps.
+
+The data is maintained in PostgreSQL. It uses [Sendgrid](https://sendgrid.com/) to send emails and does **not** currently feature an address lookup (hence no integration to OS Places).
```

Comment:
It does however consume a National Grid Reference (NGR), converts this into OS grid coords and looks up the postcode and constituency info using the external `api.postcodes.io` service